sdk/storage/azure-storage-blob/azure/storage/blob/blob_client.py

```diff
@@ -723,6 +723,7 @@ def _generic_delete_blob_options(delete_snapshots=False, **kwargs):
             delete_snapshots = DeleteSnapshotsOptionType(delete_snapshots)
         options = {
             'timeout': kwargs.pop('timeout', None),
+            'raise_on_any_failure': kwargs.pop('raise_on_any_failure', True),
             'delete_snapshots': delete_snapshots or None,
```

Comment:
This blob_client is missing the `keyword` param docstrings?

Reply:
This is an internal method.....Dont need docstrings here.